test(PlaywrightNewScreenshot): added the expectScreenshot fixture (#1192)
gravity-ui/uikit: playwright/README.md

2. Inside the component folder, create the `__tests__` folder and create a file inside it with the following name `<ComponentName>.visual.test.tsx`
3. Writing a test:

> Take a screenshot by default, takes a screenshot in a light theme

Reviewer: ```suggestion
Capture a screenshot, by default in light theme only:
```

```ts
import React from 'react';

import {expect} from '@playwright/experimental-ct-react';

> import {MyTestedComponent} from '../MyTestedComponent';

Reviewer: Just `Component` would be ok, `Tested` means it's tested?

import {test} from '~playwright/core';

> test('Name test', async ({mount}) => {

Reviewer: ```suggestion
test('test description', async ({mount}) => {
```

>   //mounting a component

Reviewer: ```suggestion
  // mount the component
```

  const component = await mount(<MyTestedComponent props={props} />);

>   //screenshot

Reviewer: ```suggestion
  // capture the screenshot
```

  await expect(component).toHaveScreenshot();
});
```

> You can also take screenshots in dark and light themes

Reviewer: ```suggestion
You can also capture screenshots both in dark and light themes:
```

```ts
import React from 'react';

import {MyTestedComponent} from '../MyTestedComponent';

import {test} from '~playwright/core';

test('Name test', async ({mount, expectScreenshot}) => {
  //mounting a component
  await mount(<MyTestedComponent props={props} />);

  //screenshot
  await expectScreenshot();
});
```

> or if you need to do any actions on the component

Reviewer: ```suggestion
If you need to do any actions with the component:
```
